src/redux-common/default-store-state.ts

```diff
@@ -17,6 +17,15 @@ export const resourceReducerDefaultState: Resources = {
   max_health: DEFAULT_MAX_HEALTH
 };
 
+export const enemyCombatantReducerDefaultState: Enemy = {
```

Comment:
Why is this in a seperate file instead of with the reducer where it's relevant ? seems confusing why you'd abstract all the default states into one file which have no relevance to eachother :S ?